### PR TITLE
Created Logout Component

### PR DESCRIPTION
**Pull request by mark-escosura**
**Created axiosWithAuth Component**
- added a function that will use axios when called along with its properties such as BaseURL, headers.

**Created a Logout Component**
- added a useEffect that removes 'token' from localStorage
- redirects into Login page

### CODE REVIEW

File: potluck/src/App.js [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
   return (
     <div className="App">
+      <nav>
+      <Link to="/logout">Logout</Link>
+      </nav>
       <Switch>
+        <PrivateRoute path="/logout">
+          <Logout />
+        </PrivateRoute>
+        <PrivateRoute path="/login">
```
Comment (ghostman1996) [inline potluck/src/App.js]: change this to a route instead of private

**Review by ghostman1996 — CHANGES_REQUESTED** [review]
change the PrivateRoute on the login page to a normal route

File: potluck/src/App.js (hunk 1 above)
Comment (mark-escosura) [reply potluck/src/App.js]: Oops.. Changed! Done.

**Review by ghostman1996 — APPROVED** [review]
Good job!

**Pull request merged**